src/gpuarray_util.c

```diff
@@ -109,6 +111,19 @@ void gpukernel_source_with_line_numbers(unsigned int count,
   }
 }
 
+static int get_flags(int typecode) {
```

Comment:
rename, there is too many things named flags. get_typecode_flags() would be better.